static/js/containers/ProfileImage.js

```diff
@@ -91,7 +95,7 @@ class ProfileImage extends React.Component {
           alt={`Profile image for ${getPreferredName(profile, false)}`}
           className="card-image"
         />
-        { this.cameraIcon(editable) }
+        { userPrivilegeCheck(profile, () => this.cameraIcon(editable) ) }
```

Comment:
what do you think about quickly refactoring `this.cameraIcon` so that it pulls `editable` out of `this.props` by itself? Then we could avoid declaring the extra function literal here, and just do

```js
{ userPrivilegeCheck(profile, this.cameraIcon) }
```